components/com_content/src/Model/FormModel.php

```diff
@@ -217,14 +217,16 @@ public function save($data)
 	 *
 	 * @param   array    $data      Data for the form.
 	 * @param   boolean  $loadData  True if the form is to load its own data (default case), false if not.
+	 * @param   boolean  $clear     Optional. Default false. True if don't want the cached form.
 	 *
 	 * @return  Form|boolean  A Form object on success, false on failure
 	 *
 	 * @since   1.6
 	 */
 	public function getForm($data = [], $loadData = true)
 	{
-		$form = parent::getForm($data, $loadData);
+		$clear = (func_num_args() > 2) ? func_get_arg(2) : false;
```

Comment:
Same as the earlier comment, add `$clear` as third option.

Reply:
This method exists since version 1.6. Many models extend it. Adding a 3rd parameter to the definition would require adding a 3rd parameter in all the child classes. And in the 3rd party extension.

So, I avoided that by using this method of accepting parameters.